pkg/util/namespace/namespace_test.go

```diff
@@ -7,62 +7,30 @@ import (
 	"testing"
 )
 
-func TestIsOpenShift(t *testing.T) {
+func TestFilteredOpenShiftNamespace(t *testing.T) {
 	for _, tt := range []struct {
 		namespace string
 		want      bool
 	}{
 		{
-			want: true,
-		},
```

Comment:
I think we still need this test case. Removing it allowed regression with non-namespaced resources to happen.